test/raven.test.js

```diff
@@ -1732,8 +1746,9 @@ describe('Raven (public API)', function() {
         it('should not throw an error if not configured', function() {
             this.sinon.stub(Raven, 'isSetup').returns(false);
             this.sinon.stub(window, 'send')
-            Raven.captureMessage('foo');
-            assert.isFalse(window.send.called);
```

Comment:
I wanted to, but I didn't manage to assert the called state on functions indirectly called by the test (maybe I just don't know the sinon test framework well enough, everytime I tried, functions indirectly called asways assert called called false, whether or now they were called).  If someone points me in the right direction, I'm quite willing to make the change.